File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/AnalyticsView.java
package openfoodfacts.github.scrachx.openfood.utils;

public enum AnalyticsView {
    SCANNER("scanner"),
    PRODUCT_EDIT_OVERVIEW("products/edit/overview"),
    PRODUCT_EDIT_INGREDIENTS("products/edit/ingredients"),
    PRODUCT_EDIT_NUTRITION_FACTS("products/edit/nutrition_facts");
    public String path;

    AnalyticsView(String path) {
        this.path = path;
    }
}

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/analytics/AnalyticsView.kt
package openfoodfacts.github.scrachx.openfood.analytics

enum class AnalyticsView(val path: String) {
    Scanner("scanner"),
    ProductEditOverview("products/edit/overview"),
    ProductEditIngredients("products/edit/ingredients"),
    ProductEditNutritionFacts("products/edit/nutrition_facts");
}
